docs/api/post-v1-querybatch.md

```diff
@@ -23,6 +23,8 @@ Query for multiple packages (by either package and version or git commit hash) a
 
 The parameters are the same as those in found [here](post-v1-query.md#parameters), but you can make multiple queries.
 
+Information on paginated results for querybatch requests is available [here](#pagination).
```

Comment:
Minor, but better not to put links under "here" --- prefer descriptive links when possible. `See [#pagination] for information on paginated results for querybatch requests.`